### PR TITLE
[14.0][MIG]account_move_print: migration from 13.0 to 14.0

### PR DESCRIPTION
**Pull request by AlvaroTForgeFlow**
@ForgeFlow

This module adds the option to print Journal Entries.

Usage
=====

To use this module, you need to:

#. Go to the *Accounting module* and click on *Accounting > Journal Entries*.
#. Create or pick one of them.
#. Inside the *print button* you will see a new option called "Journal Entries".
#. You now have your Journal Entry ready to print.

### CODE REVIEW

**Review by ChrisOForgeFlow — APPROVED** [review]
+1

File: account_move_print/report/journal_report.xml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                                    </tr>
+                                </tfooter>
+                            </table>
+                        </div>
+                    </div>
+                </t>
+            </t>
+        </t>
+    </template>
+    <report
```
Comment (ChrisOForgeFlow) [inline account_move_print/report/journal_report.xml]: This should change for new standard of reports creation

**Review by ChrisOForgeFlow — CHANGES_REQUESTED** [review]

File: account_move_print/report/journal_report.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply account_move_print/report/journal_report.xml]: @ChrisOForgeFlow suggestion done

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**